2005-05-20  Jakub Jelinek  <jakub@redhat.com>

	* configure.host (slow_pthread_self): Set to empty unconditionally
	on Linux targets.

---
 libjava/configure.host |    1 +
 1 file changed, 1 insertion(+)

Index: gcc-4.2.3/libjava/configure.host
===================================================================
--- gcc-4.2.3.orig/libjava/configure.host
+++ gcc-4.2.3/libjava/configure.host
@@ -192,6 +192,7 @@ case "${host}" in
   sh-linux* | sh[34]*-linux*)
   	can_unwind_signal=yes
 	libgcj_ld_symbolic='-Wl,-Bsymbolic'
+	slow_pthread_self=
 	if test x$slow_pthread_self = xyes \
 	   && test x$cross_compiling != xyes; then
 	  cat > conftest.c <<EOF
